src/com/goide/inspections/GoPlaceholderCountInspection.java

```diff
@@ -63,25 +74,68 @@ private static int getPlaceholderPosition(@NotNull GoFunctionOrMethodDeclaration
         return position;
       }
     }
+
     return -1;
   }
 
-  private static int getPlaceholders(@Nullable GoExpression argument) {
-    if (argument == null) return -1;
+  @Nullable
+  private static LinkedHashMap<Integer, String> getPlaceholders(@Nullable GoExpression argument) {
+    if (argument == null) return null;
 
     String placeholderText = resolve(argument);
-    if (placeholderText == null) return -1;
+    if (placeholderText == null) return null;
+
+    int placeholderTextLength = placeholderText.length();
+    int placeholderStart = 0;
+    LinkedHashMap<Integer, String> placeholders = new LinkedHashMap<Integer, String>();
+    while (placeholderStart < placeholderTextLength) {
+      ProgressManager.checkCanceled();
+      if (PLACEHOLDER_START_CHAR != placeholderText.charAt(placeholderStart)) {
```

Comment:
this actually is `placeholderText.indexOf(PLACEHOLDER_START_CHAR, placeholderStart)`